components/MailingList.jsx

```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link';
+import { FaEnvelope } from 'react-icons/fa';
+const MAILING_LIST_CONTAINER = 'bg-[#7055FD] flex flex-col text-center';
+const CONTENT_CONTAINER = 'flex flex-col pt-14 p-5 md:px-32 text-center';
```

Comment:
could you ever so slightly reduce the top padding? On mobile view the gap at the top looks pretty big compared to on desktop. I think you could just do something like `py-5` so it's even on the top and bottom
<img width="402" alt="Screenshot 2024-01-23 at 6 05 05 PM" src="https://github.com/Code-the-Change-YYC/code-the-change-yyc-site/assets/65151396/b3686e79-ab02-491f-b23f-fef9fdb5bc46">